packages/database/schema.prisma

```diff
@@ -318,6 +318,7 @@ model Survey {
   displayPercentage  Decimal?
   languages          SurveyLanguage[]
   showLanguageSwitch Boolean?
+  endings            Json[]           @default([])
```

Comment:
please move `endings` up below the `questions` so also in prisma studio this shows after the questions which makes it easier to manually make changes to records.